### PR TITLE
fix: add outside customization to some components

### PR DESCRIPTION
**Pull request by gerardparareda**
## Changes
- Changed function `onRowStyle` from required to optional.
- Added CSS-in-JS in the HTML `customStyle` parameter some of the following components:
 - Container (because of the styled component I used the https://www.npmjs.com/package/style-object-to-css-string library)
 - Table
 - Headers
 - Cells

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,8 @@
   "dependencies": {
     "react": "^16.8.0 || 17.x",
     "react-dom": "^16.8.0 || 17.x",
+    "react-jss": "^10.10.0",
```
Comment (mguellsegarra) [inline package.json:16]: @gerardparareda I think that this might be an unused dependency, am I mistaken? 😕

Comment (gerardparareda) [reply package.json:16]: Nope, you are completely right. I missed this!

**Pull request merged**